server/app/Http/routes.php

```diff
@@ -263,24 +219,17 @@
         $feed->lang = 'en';
 
         foreach ($docs as $doc) {
-            $sponsor = $doc->sponsor->first();
-            if ($sponsor instanceof User) {
-                $display_name = $sponsor->fname.' '.$sponsor->lname;
-            } elseif ($sponsor instanceof Group) {
-                $display_name = $sponsor->display_name;
-            } else {
-                $display_name = '';
-            }
-
-            $item = array();
+            $item = [];
             $item['title'] = $doc->title;
-            $item['author'] = $display_name;
-            $item['link'] = URL::to('docs/'.$doc->slug);
+            $item['author'] = $doc->sponsors->first()->display_name;
+            $item['link'] = $doc->url;
             $item['pubdate'] = $doc->updated_at;
             $item['description'] = $doc->title;
-            $item['content'] = $doc->content->html();
+            $item['content'] = $doc->fullContentHtml();
+            $item['enclosure'] = [];
+            $item['category'] = '';
 
-            array_push($feed->items, $item);
+            $feed->addItem($item);
         }
```

Comment:
Also this should use the cache functionality and invalidate on document events.  No reason to generate this every time.